.changeset/lucky-brooms-shake.md

```diff
@@ -0,0 +1,5 @@
+---
+'@reown/appkit-scaffold-ui': patch
```

Comment:
@mrcattington can you add all packages here? When creating changeset with `pnpm changeset` script, you should select all packages.

Also follow convention like:

"Updates legal links to open them in new tab"

Reply:
https://github.com/reown-com/appkit/pull/4629
After this it shouldn't matter 🫡